File: src/1550.three-consecutive-odds.1293/solution.js
/**
 * @param {number[]} arr
 * @return {boolean}
 */
var threeConsecutiveOdds = function (arr) {
    for (let i = 0; i < arr.length - 2; i++) {
        if (arr[i] % 2 === 1 && arr[i + 1] % 2 === 1 && arr[i + 2] % 2 === 1) {
            return true;
        }
    }
    return false;
};

File: src/1550.three-consecutive-odds.1293/solution.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        int index = 0;
        while (index<arr.length){
            if(arr[index++]%2 == 0){
                continue;
            }
            
            int cnt = 1;
            while(index<arr.length && arr[index]%2 == 1){
                cnt++;
                index++;
            }
            if(cnt>2){
                return true;
            }
        }
        return false;
    }
}